waveform_collection/server.py

```diff
@@ -74,26 +74,31 @@ def gather_waveforms(source, network, station, location, channel, starttime,
             ``http://10.30.5.10:8080`` if using VPN)
         watc_username (str): Username for WATC FDSN server
         watc_password (str): Password for WATC FDSN server
+        verbose (bool): If `False`, all print statements will be blocked. 
+            Default is `True`.
 
     Returns:
         :class:`~obspy.core.stream.Stream` containing gathered waveforms. If
         `return_failed_stations` is `True`, additionally returns a list
         containing station codes that were requested but not downloaded
     """
-
+    
```

Comment:
Make sure your editor is removing all whitespace! I have mine configured to do this on save. This will keep our diffs smaller. :)
```suggestion

```